MariaDB.md

```diff
@@ -0,0 +1,199 @@
+## WAL-G for MariaDB
```

Comment:
There is no point in copy-pasting MySQL documentation.
It will require twice more efforts to maintain it.

The real things needs to be done with MariaDB are:
* run tests agains MariaDB image
* write more test about PITR, as it relies heavily on GTIDs and MariaDB has different GTID format.

Reply:
Should I create a different PR for tests?
> It will require twice more efforts to maintain it.
Initial lazyness, I will merge the new bits with the original one in a single file.

Reply:
I've merged the two files together by adding the mariadb extra bits at the end of mysql.md.
I'm not sure about the replication bit, should I just refer the reader to the docs and point out the differences to mysql, or should include the correct steps?
I have never done replication with mysql/mariadb before I'm still trying to understand how the restore procedure works in case of replications + wal-g.

Reply:
You could add tests in this PR.
Thank you !

Reply:
Lucky day @yuk1ru has already created half of the tests. Will work with him on them.